apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -115,7 +93,43 @@ export class DotEditLayoutComponent implements OnInit, OnDestroy {
             .subscribe(
                 (updatedPage: DotPageRender) => this.handleSuccessSaveTemplate(updatedPage),
                 (err: ResponseView) => this.handleErrorSaveTemplate(err),
-                () => (this.didTemplateChanged = false)
+                () => this.canRouteBeDesativated(true)
+            );
+    }
+
+    /**
+     *  The reason why we are using a Subject [updateTemplate] here is
+     *  because we can not just simply add a debounceTime to the HTTP Request
+     *  we need to reset the time everytime the observable is called.
+     *
+     *  More Information Here:
+     *          - https://stackoverflow.com/questions/35991867/angular-2-using-observable-debounce-with-http-get
+     *          - https://blog.bitsrc.io/3-ways-to-debounce-http-requests-in-angular-c407eb165ada
+     *
+     * @memberof DotEditLayoutComponent
+     */
```

Comment:
This is not doc, the is is a comment inside the method. If we leave this here it will show up in the generated documentation.